css/style.css

```diff
@@ -0,0 +1,308 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body,
+html {
+  overflow-x: hidden;
+}
+
+.up-header {
+  display: none;
+}
+
+.font {
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+}
+
+#navbarNavDropdown {
+  display: none;
+  background-color: #333;
+  width: 75%;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 2;
+}
+
+#navbarNavDropdown a {
+  color: #fff;
+  font-family: 'cocogoose', sans-serif;
+  font-size: 20px;
+  text-decoration: none;
+  margin: 3px 50px;
+}
+
+#nav-btn {
+  border: 4px solid #ec5242;
+  margin-left: 5px;
+}
+
+.color {
+  color: #272a31;
+}
+
+.color1 {
+  color: #ec5242;
+}
+
+.color2 {
+  color: #d3d3d3;
+}
+
+.no-scroll {
+  overflow: hidden;
+}
+
+.active {
+  color: #ec5242;
+}
+
+.first-container {
+  background-image: url('../images/bgimgfinal.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 100%;
+  text-align: center;
+  padding: 30px;
+}
+
+.font-bold {
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+}
+
+footer img {
+  width: 100px;
+  margin: 20px 40px;
+}
+
+.first-container .information {
+  justify-content: center;
+}
+
+.first-container .title {
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+}
+
+.first-container .heading1 {
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+}
+
+.first-container .event-details {
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  color: #272a31;
+}
+
+.first-container .event-details .first-para {
+  border: 5px solid white;
+  font-size: 18px;
+}
+
+.first-container .info-home {
+  color: #ec5242;
+}
+
+.first-container .info-home .heading1 {
+  font-family: 'cocogoose', sans-serif;
+  font-size: 60px;
+}
+
+.first-container .event-details h3 {
+  font-family: 'Lato', sans-serif;
+  font-weight: bold;
+  font-size: 30px;
+}
+
+.first-container .event-details .venue {
+  font-size: 20px;
+}
+
+.up-footer {
+  height: 250px;
+}
+
+.down-footer {
+  background: #d3d3d3;
+}
```

Comment:
- [ ] The footer's background should be white on the Mobile version of the page and dark on the desktop version of the page. Look at the screenshot below to help you notice the differences and make necessary adjustments. Take care of the font colors too.

| The Provide Design |
| --- |
| ![footer-desktop-design](https://user-images.githubusercontent.com/45002654/133670516-d031cba3-60f2-4ab6-80cc-4a8dabdf53fd.PNG) |

| Your implementation |
| --- |
| ![footer-imp](https://user-images.githubusercontent.com/45002654/133670633-260919a4-1a20-47a0-999d-012691f8fe97.PNG) |